From f86dfffa8d443ec640b1c76d2eeccd6a4913305d Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@gmail.com>
Date: Fri, 17 Jan 2025 15:32:51 +0000
Subject: [PATCH] build: drop man directory

The man pages require rst2man, which isn't available (or at least
not trivial to provide), so we drop the man directory to avoid
building the pages.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/meson.build b/meson.build
index 8862ada8f..22a14d53f 100644
--- a/meson.build
+++ b/meson.build
@@ -15,7 +15,6 @@ pkgdir = datadir / 'virt-manager'
 git = run_command('test', '-e', '.git', check: false).returncode() == 0
 
 subdir('scripts')
-subdir('man')
 subdir('data')
 subdir('ui')
 subdir('virtinst')
-- 
2.39.2

